Rules/Playbooks/Thief.md

```diff
@@ -0,0 +1,33 @@
+
+You gain XP when you roll a desperate action (in that action attribute only). At the end of the session, increase any XP clock if...
+- You express your beliefs, drives, heritage, or background
+- You struggled with your vice or traumas
+- You addressed a challenge with stealth or evasion
+## Items
+
+- fine thieves tools
+- [ ] fine pressure suit
+- [ ] infrared goggles
+- implanted field manipulator
+- confusion spores
+- [ ] ion scooter
```

Comment:
I'm imagining scuba scooters in space here. "Ion" referring to [ion thrusters](https://en.wikipedia.org/wiki/Ion_thruster) because they sound scifi and have high SI